packages/@webex/plugin-meetings/src/roap/turnDiscovery.ts

```diff
@@ -202,6 +202,28 @@ export default class TurnDiscovery {
    * @returns {Promise}
    */
   doTurnDiscovery(meeting, isReconnecting) {
+    const reachabilityData = window.localStorage.getItem(REACHABILITY.localStorage);
```

Comment:
It would be nice to have this whole new block of code encapsulated into a function, something like `isReachabilitySuccessful()` or something like that. Ideally this function should be inside the src/reachability/index.js so that all reachability specific logic is in 1 place.